Tests/UI/New UI/BillingForm/View/BillingFormSummaryViewTests.swift

```diff
@@ -46,7 +46,7 @@ class BillingFormSummaryViewTests: XCTestCase {
     }
 
     func testStyleIconImageView() {
-        XCTAssertEqual(view.imageContainerView?.imageView?.image, style.button.image)
+        XCTAssertEqual(view.imageContainerView?.imageView?.image, style.button.image?.imageFlippedForRightToLeftLayoutDirection())
```

Comment:
Question : Can you compare 2 images ? this way. Will this even be a valid test.

Reply:
I think, It's valid test. this test is against the the provided image style.